firmware/application/apps/ais_app.hpp

```diff
@@ -127,115 +119,94 @@ class AISRecentEntryDetailView : public View {
-    AISRecentEntryDetailView &operator=(const AISRecentEntryDetailView&Entry);
+	AISRecentEntryDetailView(const AISRecentEntryDetailView& Entry);
+	AISRecentEntryDetailView& operator=(const AISRecentEntryDetailView& Entry);
 
-private:
-	AISRecentEntry entry_ { };
+ private:
+	AISRecentEntry entry_{};
 
-	Button button_done {
-		{ 125, 224, 96, 24 },
-		"Done"
-	};
-	Button button_see_map {
-		{ 19, 224, 96, 24 },
-		"See on map"
-	};
-	GeoMapView* geomap_view { nullptr };
-	bool send_updates { false };
-
-	Rect draw_field(
-		Painter& painter,
-		const Rect& draw_rect,
-		const Style& style,
-		const std::string& label,
-		const std::string& value
-	);
+	Button button_done{{125, 224, 96, 24}, "Done"};
+	Button button_see_map{{19, 224, 96, 24}, "See on map"};
+	GeoMapView* geomap_view{nullptr};
+	bool send_updates{false};
+
+	Rect draw_field(Painter& painter,
+									const Rect& draw_rect,
+									const Style& style,
+									const std::string& label,
+									const std::string& value);
 };
 
 class AISAppView : public View {
-public:
+ public:
 	AISAppView(NavigationView& nav);
 	~AISAppView();
 
 	void set_parent_rect(const Rect new_parent_rect) override;
 
 	// Prevent painting of region covered entirely by a child.
-	// TODO: Add flag to View that specifies view does not need to be cleared before painting.
-	void paint(Painter&) override { };
+	// TODO: Add flag to View that specifies view does not need to be cleared
+	// before painting.
+	void paint(Painter&) override{};
 
 	void focus() override;
 
 	std::string title() const override { return "AIS RX"; };
 
-private:
+ private:
 	static constexpr uint32_t initial_target_frequency = 162025000;
 	static constexpr uint32_t sampling_rate = 2457600;
 	static constexpr uint32_t baseband_bandwidth = 1750000;
 
-
 	// app save settings
-	std::app_settings 		settings { }; 		
-	std::app_settings::AppSettings 	app_settings { };
+	std::app_settings settings{};
+	std::app_settings::AppSettings app_settings{};
 
 	NavigationView& nav_;
 
-	AISRecentEntries recent { };
-	std::unique_ptr<AISLogger> logger { };
+	AISRecentEntries recent{};
+	std::unique_ptr<AISLogger> logger{};
 
-	const RecentEntriesColumns columns { {
-		{ "MMSI", 9 },
-		{ "Name/Call", 20 },
-	} };
-	AISRecentEntriesView recent_entries_view { columns, recent };
-	AISRecentEntryDetailView recent_entry_detail_view { nav_ };
+	const RecentEntriesColumns columns{{
+			{"MMSI", 9},
+			{"Name/Call", 20},
+	}};
+	AISRecentEntriesView recent_entries_view{columns, recent};
+	AISRecentEntryDetailView recent_entry_detail_view{nav_};
 
 	static constexpr auto header_height = 1 * 16;
 
-	Text label_channel {
-		{ 0 * 8, 0 * 16, 2 * 8, 1 * 16 },
-		"Ch"
-	};
+	Text label_channel{{0 * 8, 0 * 16, 2 * 8, 1 * 16}, "Ch"};
 
-	OptionsField options_channel {
-		{ 3 * 8, 0 * 16 },
-		3,
-		{
-			{ "87B", 161975000 },
-			{ "88B", 162025000 },
-		}
-	};
+	OptionsField options_channel{{3 * 8, 0 * 16},
```

Comment:
I wonder if there's a bug in the parser? It seems to keep doing this particular strange case.

Reply:
Line length

Reply:
Maybe the diff is busted, but this is what I see (here and in the others):
<img width="1168" alt="image" src="https://github.com/eried/portapack-mayhem/assets/21130052/d61227e8-2899-4793-ae0b-d2897bee91f3">

Reply:
I'll have to check it out tomorrow when I get back as my screen too small on my phone